src/slack/routes.js

```diff
@@ -27,18 +30,40 @@ router.post('/slack/echo', (request, response) => {
 
 router.post('/slack/inspireme', (request, response) => {
   const channelName = request.body.channel_name;
+  try {
```

Comment:
Currently our routes call functions from a `resourceServer` module.

In this case the `/slack/inspire-me` route expects a function with the following signature:
`resourceServer.getInspiration(userId) -> {Object}`